<tempfile._TemporaryFileWrapper object at 0x1819803dd8>

```diff
@@ -0,0 +1 @@
                                                                                                                        XTENSION= 'TABLE   '           / ASCII table extension                          BITPIX  =                    8 / array data type                                NAXIS   =                    2 / number of array dimensions                     NAXIS1  =                    0 / length of dimension 1                          NAXIS2  =                    0 / length of dimension 2                          PCOUNT  =                    0 / number of group parameters                     GCOUNT  =                    1 / number of groups                               TFIELDS =                    0 / number of table fields                         VERSION =                    1                                                  EXTNAME = 'GLOBAL  '           / extension name                                 END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
```

Comment:
I think this tempfile should not be part of this PR?

Reply:
oops, yes that was mistakenly committed. I'll remove it.

Reply:
also, it seems to be a bug I introduced - the object name is being used as the filename...